data/js/utility/content3.js

```diff
@@ -1,12 +1,10 @@
 (function() {
     // retrieve our default text from the sidebar
     var defaultText = document.querySelector('#default');
-    var timeout;
-    // set template and hoveremit names
+    // set template names
     var templateNames = ['template1','template2','template3'];
-    var hoverEmit = ['hoverTemplate1','hoverTemplate2','hoverTemplate3'];
 
-    // add listeners to all of our buttons for hovering and clicking
+    // add listeners to all of our buttons for clicking
     for(var x=0; x<templateNames.length; x++)
     {
```

Comment:
Nit: For consistent code style do `for (var x = 0....) {` i.e. opening curly brace on same line as `for` statement. Also, cache the array length. This improves performance and is a general best practice:

```
for (var i = 0, l = templateNames.length; i < l; i++) {
```